src/ui.rs

```diff
@@ -280,7 +280,7 @@ fn display_message(
 
     let idx = names_and_colors
         .binary_search_by_key(&msg.from_id, |&(id, _, _)| id)
-        .unwrap();
+        .unwrap_or_default();
```

Comment:
In case the binary search did not find the user, the `unwrap_or_default()` returns 0, that is, just uses the name of the first user in the `name_and_colors` vec.

I guess, it would be better to get users name with `app.name_by_id` and some hard coded color for the users that do not exist.

Reply:
Perfect!